src/serialization/compact/SchemaService.ts

```diff
@@ -106,4 +129,42 @@ export class SchemaService {
         const invocation = new Invocation(this.getInvocationService(), message);
         return this.getInvocationService().invokeUrgent(invocation).then(() => {});
     }
+
+    replicateSchemaInCluster(schema: Schema): Promise<boolean> {
+        const clientMessage = ClientSendSchemaCodec.encodeRequest(schema);
+        return this.retryMaxPutRetryCount(clientMessage, 0);
+    }
+
+    private retryMaxPutRetryCount(clientMessage: ClientMessage, index: number) : Promise<boolean> {
```

Comment:
```suggestion
    private retryMaxPutRetryCount(clientMessage: ClientMessage, currentRetryCount: number) : Promise<boolean> {
```

or it can be `retryCount`. IMO, index is too general

Reply:
Fixed